app/models/setting/katello.rb

```diff
@@ -14,7 +14,8 @@ def self.load_defaults
         self.set('content_action_finish_timeout', N_("Time in seconds to wait for a Host to finish a remote action"), 3600),
         self.set('pulp_sync_node_action_accept_timeout', N_("Time in seconds to wait for a pulp node to remote action"), 20),
         self.set('pulp_sync_node_action_finish_timeout', N_("Time in seconds to wait for a pulp node to finish sync"), 12.hours.to_i),
-        self.set('check_services_before_actions', N_("Whether or not to check the status of backend services such as pulp and candlepin prior to performing some actions."), true)
+        self.set('check_services_before_actions', N_("Whether or not to check the status of backend services such as pulp and candlepin prior to performing some actions."), true),
+        self.set('force_post_sync_actions', N_("Force post sync actions such as indexing and email even if no content was available."), false)
```

Comment:
Might be nice to state what kind of emails here. The term of just email had me a bit baffled until I remember I think we send out errata applicability emails.